src/helpers/AddressHelpers.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Formats addresses as [CAIP-10](https://chainagnostic.org/CAIPs/caip-10) addresses for it's respective request scope. See [CAIP-25](https://github.com/ChainAgnostic/CAIPs/blob/main/CAIPs/caip-25.md) and .
+ * @param scope - The scope to create session for.
+ * @param addresses - The addresses to format. If address is empty, we remove it from the array.
+ * @returns The formatted addresses with the scope to create session for.
+ */
+export const getCreateSessionOptionalScopesFormattedAddresses = (
+  scope: string,
+  addresses: string[],
+): string[] => {
+  return addresses.reduce<string[]>((result, address) => {
+    if (address.length > 0) {
+      result.push(`${scope}:${address}`);
+    }
+    return result;
+  }, []);
+};
```

Comment:
I considered adding steps to assert passed addresses are valid EVM addresses (ex.:`0x81ab...007`), but ticket scope says `"We'll need to interate on this later for non-evm but for now assume all addresses are EVM."`, so we can skip verification ?